test/integration/integration.cc

```diff
@@ -409,10 +409,25 @@ void BaseIntegrationTest::setUpstreamAddress(
   socket_address->set_port_value(fake_upstreams_[upstream_index]->localAddress()->ip()->port());
 }
 
+namespace {
+bool read_flag(bool* flag) { return *flag; }
```

Comment:
We can merge master once the other PR is reviewed and merged.

Reply:
Of course. Early merge it here only to prove this PR is well tested.